doc/create-cluster-kubectl.md

```diff
@@ -0,0 +1,72 @@
+# Use Exoscale ClusterAPI
+
+## Create bootstrap cluster
+
+### Use an existing k8s cluster or bootstrap it
+
+With [kind](https://github.com/kubernetes-sigs/kind):
+```
+% kind create cluster --name=clusterapi
+```
+```
+% export KUBECONFIG="$(kind get kubeconfig-path --name="clusterapi")"
+```
+
+## Deploy custom resources
+run `make`
+
+```
+% ./bin/clusterctl --kubeconfig=$KUBECONFIG  \
+                   alpha phases \
+                   apply-cluster-api-components \
+                   -p provider-components.yaml
```

Comment:
or maybe just a `kubectl apply -f provider-components.yaml`

Reply:
@greut and `clusterctl` in this case is not needed

Reply:
I'll have to dig the documentation to see what's behind the "apply-cluser-api-components" ;)

Reply:
Thanks :smile:

Reply:
I'll try ASAP